src/vit-based/vit.jl

```diff
@@ -80,7 +82,7 @@ Creates a Vision Transformer (ViT) model.
 - `pool`: pooling type, either :class or :mean
 - `nclasses`: number of classes in the output
 
-See also [`Metalhead.vit`](#).
+See also [`Metalhead.vit`](@ref) for a more flexible constructor.
```

Comment:
I'm not sure what `#` was for

Reply:
That's the syntax used by Publish/Pollen.